test/metabase/automagic_dashboards/core_test.clj

```diff
@@ -113,6 +112,46 @@
       (with-dashboard-cleanup
         (-> card-id Card (automagic-analysis {}) valid-dashboard?)))))
 
+(expect
+  true
+  (tt/with-temp* [Card [{card-id :id} {:table_id      nil
```

Comment:
You don't need to put `true` here because `expect` with 1 arg assumes `expected` is `true`.